src/components/Widget/components/Conversation/index.tsx

```diff
@@ -1,13 +1,28 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import * as React from 'react';
 
 import Header from './components/Header';
 import Messages from './components/Messages';
 import Sender from './components/Sender';
 import QuickButtons from './components/QuickButtons';
 import './style.scss';
 
-const Conversation = props =>
+interface IConversation {
+  title: string;
+  titleAvatar: string;
+  subtitle: string;
+  sendMessage: Function;
+  senderPlaceHolder: string;
+  profileAvatar: string;
+  toggleChat: Function;
+  showCloseButton: boolean;
+  disabledInput: boolean;
+  autofocus: boolean;
+  onQuickButtonClicked: Function;
+  showChat: boolean;
+  className: string;
+};
+
+const Conversation = (props: IConversation) =>
```

Comment:
I'd do `const Conversation = ({ /* destructure props */ } : IConversation)` so we don't have to do `props.` all the time.